### ISSUE TITLE
launchIDE requires the project to compile

### ISSUE THREAD

**Issue by Blaisorblade** [lampepfl/dotty#4469]
I like to use an IDE to fix my type errors. This also requires the project to fully compile, not just to compile with the presentation compiler, so it will fail even for code that passes the presentation compiler and/or can produce Tasty (but fails later).

Since I see why having some compiled output helps, I'm not sure what the architecture should be here.

**Comment by smarter** [comment]
Compiling before starting the IDE hasn't been strictly necessary for a couple of months now (if a classfile is older than a corresponding source file, the source file will be typechecked), but it's still useful to have classfiles for everything for performance in the IDE. It should be possible to make launchIDE compile the project, and then start the IDE even if it doesn't compile.

**Comment by Blaisorblade** [comment]
But should the IDE use classfiles or tasty, long term? The latter could give superior results, but I don't know how it relates to the current roadmap.

**Comment by smarter** [comment]
When I say classfiles I mean the tasty part of classfiles.

**Comment by nicolasstucki** [comment]
This is a duplicate of #3114
